client/layout/guided-tours/tours/jetpack-plugin-updates-tour.js

```diff
@@ -0,0 +1,91 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import { query } from 'layout/guided-tours/positioning';
+
+export const JetpackPluginUpdatesTour = makeTour(
+	<Tour name="jetpackPluginUpdates" version="20180611">
+		{ /* Phantom step, wait for placeholder to disappear, then advance */ }
+		<Step
+			name="init"
+			target=".plugin-item.is-placeholder"
+			onTargetDisappear={ ( { next } ) => next() }
+			next="onLoaded"
+		/>
```

Comment:
Magic, exhibit 1 (wait for placeholders to disappear)…

Reply:
:clap: Well played, Jon.